app/build.gradle

```diff
@@ -72,6 +72,9 @@ android {
 
       debug {
          ext.enableCrashlytics = false
+         applicationIdSuffix ".debug"
+         resValue "string", "authority", applicationIdSuffix + '.data.QuranDataProvider'
+         resValue "string", "file_authority", applicationIdSuffix + '.fileprovider'
```

Comment:
not sure if these are right. these should be whatever the package name is plus the suffix so that if you build qaloonDebug, you get the right package name for example (probably have to try it and see)